R/etwfe.R

```diff
@@ -77,6 +77,8 @@ etwfe = function(
     gvar = NULL,
     data = NULL,
     ivar = NULL,
+    intvar = NULL,
+    id   = NULL,
```

Comment:
Can we use a different set of argument names? For consistency with the other arguments, I'd prefer something like `xvar` and `xref`.

Also: You need to document the arguments as part of the roxygen2 preamble, otherwise it's going to generate a bunch of CRAN check fails. See the top of the doc for examples.